packages/web/components/Player/Box/PlayerGallery.tsx

```diff
@@ -1,63 +1,149 @@
-import { Box, Flex, HStack, Text } from '@metafam/ds';
+import {
+  Box,
+  Flex,
+  HStack,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+  SimpleGrid,
+  Text,
+  useDisclosure,
+} from '@metafam/ds';
+import { MetaLink as Link } from 'components/Link';
+import { PlayerFragmentFragment } from 'graphql/autogen/types';
+import {
+  Collectible,
+  useOpenSeaCollectibles,
+} from 'lib/useOpenSeaCollectibles';
 import React from 'react';
+import { FaTimes } from 'react-icons/fa';
 
-import nft1Image from '../../../assets/fake/nft1.png';
-import nft2Image from '../../../assets/fake/nft2.png';
 import { PlayerBox } from './PlayerBoxe';
 
-// TODO Fake data
-type Props = { setRemoveBox: () => void };
-export const PlayerGallery: React.FC<Props> = ({ setRemoveBox }) => {
-  const [show, setShow] = React.useState(false);
-  const fakeEthPrice = 500;
-  const fakeData = [
-    { title: 'CryptoMon Cards - Aave', priceInEth: 0.025, img: nft1Image },
-    { title: 'metagamer', priceInEth: 6.942, img: nft2Image },
-  ];
+type Props = { player: PlayerFragmentFragment; setRemoveBox: () => void };
+
+const GalleryItem: React.FC<{ nft: Collectible; noMargin?: boolean }> = ({
+  nft,
+  noMargin = false,
+}) => (
+  <Link
+    href={nft.permaLink}
+    isExternal
+    mb={noMargin ? undefined : 6}
+    minW={0}
+    display="flex"
+  >
+    <HStack spacing={6}>
+      <Flex width="126px" height="126px">
```

Comment:
Sizing is hard coded as a magic number. Can we use responsive styles or one of the standard sizes from the theme instead for consistency: https://chakra-ui.com/docs/theming/theme#sizes ?